README.adoc

```diff
@@ -20,6 +20,22 @@ image:https://badges.gitter.im/eclipse/microprofile-reactive.svg[link="https://g
 
 = MicroProfile Reactive Streams Operators
 
+'''
+== How to contribute
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
+* https://github.com/eclipse/microprofile-reactive-streams-operators/issues[Create or fix an issues].
+* https://gitter.im/eclipse/microprofile-reactive[Join us on Gitter to discuss this project].
+* Join our https://calendar.google.com/calendar/embed?src=gbnbc373ga40n0tvbl88nkc3r4%40group.calendar.google.com[bi-weekly meeting] every second Tuesday at https://www.timeanddate.com/time/map/[10h00 GMT]. 
```

Comment:
The reactive hangouts are every week. So probably the wrong link.

Reply:
Fixed. Will commit soon.

Reply:
Done